src/runtime/ProtocolBuffers.xcodeproj/project.pbxproj

```diff
@@ -1208,9 +1193,10 @@
 				HEADER_SEARCH_PATHS = ../ProtocolBuffers;
 				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
 				OBJROOT = Build;
+				ONLY_ACTIVE_ARCH = NO;
 				OTHER_LDFLAGS = "-ObjC";
 				PRIVATE_HEADERS_FOLDER_PATH = ../ProtocolBuffers/private;
-				PUBLIC_HEADERS_FOLDER_PATH = ../ProtocolBuffers;
+				PUBLIC_HEADERS_FOLDER_PATH = ProtocolBuffers;
```

Comment:
this path does not exist .. ?

Reply:
It's easier for the dependent components to find the header files. Also I noted that by default when creating a static library with Xcode, the public headers go under $(CONFIGURATION_BUILD_DIR) (which expands to for example Build/Products/Release-iphoneos) and therefore I was thinking to do it here too. We don't need to change the public header folder path though, I can workaround that and perhaps it's better since this is a fork and less changes we make easier is to maintain it.